lib/handlers/injector-sor.ts

```diff
@@ -467,14 +468,26 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
             tenderlySimulator,
             ethEstimateGasSimulator
           )
+          const newCachedRoutesRolloutPercent = NEW_CACHED_ROUTES_ROLLOUT_PERCENT[chainId]
 
           let routeCachingProvider: IRouteCachingProvider | undefined = undefined
-          if (CACHED_ROUTES_TABLE_NAME && CACHED_ROUTES_TABLE_NAME !== '') {
-            routeCachingProvider = new DynamoRouteCachingProvider({
-              routesTableName: ROUTES_TABLE_NAME!,
-              routesCachingRequestFlagTableName: ROUTES_CACHING_REQUEST_FLAG_TABLE_NAME!,
-              cachingQuoteLambdaName: AWS_LAMBDA_FUNCTION_NAME!,
-            })
+
+          if (Math.random() * 100 < (newCachedRoutesRolloutPercent ?? 0)) {
+            if (CACHED_ROUTES_TABLE_NAME && CACHED_ROUTES_TABLE_NAME !== '') {
```

Comment:
nit: you can move this line outside - common in both cases

Reply:
yea, seems more readable that way, will do